web/src/components/layout/Icon.jsx

```diff
@@ -149,10 +147,19 @@ const icons = {
  *
  */
 export default function Icon({ name, className = "", size = 32, ...otherProps }) {
+  if (!icons[name]) {
+    console.error(sprintf(_("Icon %s not found!"), name));
```

Comment:
NOTE: I wonder if worth it adding a `process.env.NODE_ENV` check for not delivering these `console.error` in the production build.

```suggestion
    process.env.NODE_ENV !== "production" && console.error(sprintf(_("Icon %s not found!"), name));
```

or even to have a `logError` util for using it in every place we do a console.error

```js
const logError = (error) => {
  if (process.node.NODE_ENV !== "production") {
    console.error(error);
  }
};
```


```suggestion
    logError(sprintf(_("Icon %s not found!"), name));
```

Not sure if webpack automatically drop these blocks from the production build. Has to check.

https://webpack.js.org/guides/production/#specify-the-mode

Reply:
I've checked by my own that blocks conditioned to `process.env.NODE_ENV !== 'production'` are not shipped in the production build. Which is great. So, creating an issue for wrap all `console.error`s

Reply:
Discarded. See https://github.com/openSUSE/agama/issues/842